packages/expo-document-picker/src/types.ts

```diff
@@ -12,6 +12,7 @@ export type DocumentPickerOptions = {
    * which allows other Expo APIs to read the file immediately. This may impact performance for
    * large files, so you should consider setting this to `false` if you expect users to pick
    * particularly large files and your app does not need immediate read access.
+   * @platform ios,android
```

Comment:
```suggestion
   * @platform ios
   * @platform android
```

Separating platforms with a comma using a single `@platform` tag will result in the following:

![CleanShot 2025-06-16 at 16 30 28@2x](https://github.com/user-attachments/assets/8c4eae46-b9ab-4aa3-8e46-006e90c6e1b7)

Let's use separate platform tags for each.

Reply:
Thank you, fixed